### ISSUE TITLE
Unable to hit breakpoint in JS in a Blazor WebAssembly app

### ISSUE THREAD

**Issue by danroth27** [dotnet/aspnetcore#20118]
Repro:
- Create a Blazor WebAssembly app
- Add a Razor Class Library to the solution and reference it
- Update *Index.razor* to call the `Prompt` method:

```razor
@page "/"
@using RazorClassLibrary1
@inject IJSRuntime JS

<h1>Hello, world!</h1>

Welcome to your new app.

<SurveyPrompt Title="How is Blazor working for you?" />

<button style="@buttonStyle" class="btn btn-primary" @onclick="OnClick">Prompt</button>

@code {
    string buttonStyle;

    async Task OnClick()
    {
        var color = await ExampleJsInterop.Prompt(JS, "What is your favorite color?");
        buttonStyle = $"background: {color}";
    }
}
```

- Reference *exampleJsInterop.js* from *index.html*

```
<script src="_content/RazorClassLibrary/exampleJsInterop.js"></script>
```

- Set a breakpoint in *exampleJsInterop.js*
- F5 to debug the app and hit the Prompt button

Expected result:
- Breakpoint hit in *exampleJsInterop.js*

Actual result:
- Breakpoint doesn't get hit and the breakpoint shows up as unbound in VS:

![image](https://user-images.githubusercontent.com/1874516/77490828-4af3d700-6df9-11ea-93a5-d26ab8fc7957.png)

Workaround:
- Expand the Script Documents node in VS, open exampleJsInterop.js from the list, and set the breakpoint there.
- The breakpoint will then hit, but still shows up as unbound:

![image](https://user-images.githubusercontent.com/1874516/77490889-7bd40c00-6df9-11ea-8604-d19ff18e77b6.png)

**Comment by mkArtakMSFT** [comment]
The work here is actually on the VS side and the pending fix is being tracked internally as part of https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1107462

**Comment by mkArtakMSFT** [comment]
Glad you've found a workaround, @Daddoon. We are working on a fix and it'll be available when Blazor WASM goes to GA (very soon).

**Comment by mkArtakMSFT** [comment]
This is now resolved and will be available in the VS 16.6 Preview 5 release.
